recipes/symopt/meta.yaml

```
@@ -0,0 +1,54 @@
+{% set name = "symopt" %}
+{% set version = "0.1.0" %}
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  url: https://github.com/spcornelius/{{ name }}/archive/v{{ version }}.tar.gz
+  sha256: 24335e1ed3306a5b30f9f7eab97b8bb66b562c625a260c32d0f8caaf2a485ec9
+
+build:
+  skip: True  # [win]
+  number: 0
+  script: python -m pip install . --no-deps -vv
+
+requirements:
+  build:
+    - python >= 3.6
```

There should be a space before the version constraint but not before the number:
```suggestion
    - python >=3.6
```
(`conda-build` doesn't enforce this everywhere so builds work like this by feedstock creation doesn't)